config/health_care_application/wsdl/voa.wsdl

```diff
@@ -3,7 +3,6 @@
 -->
 <definitions
     xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/"
-    xmlns:tns="http://va.gov/service/esr/voa/v1"
```

Comment:
removing this header/namespace (duplicated in `targetNamespace`) produced the same successful request/response as the old version of savon.